test/fast/Running 'nvm-exec' should display required node version

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+set -x
+\. ../../nvm.sh
+
+NVM_TEST_VERSION=v0.42
+
+# Write it to nvmrc
+echo "$NVM_TEST_VERSION" > .nvmrc
+
+OUTPUT="$(../../nvm-exec 2>&1)";
+EXPECTED="N/A: version \"${NVM_TEST_VERSION}\" is not yet installed.
+
+You need to run \`nvm install ${NVM_TEST_VERSION}\` to install and use it.
+No NODE_VERSION provided; no .nvmrc file found";
+
+# Skip install, we want to test the error message
+diff <(echo "${EXPECTED}") <(echo "${OUTPUT}")
```

Comment:
this line can't work tho, due to https://www.shellcheck.net/wiki/SC3001 - i'll change this to a normal `=` comparison unless you have a better idea

Reply:
Yeah, sound reasonable.
Was hoping that the `#!/bin/bash` on top would allow for that to pass, but I think `=` should work too.